src/sinks/gcp/stackdriver_metrics.rs

```diff
@@ -110,6 +110,10 @@ impl SinkConfig for StackdriverConfig {
     fn sink_type(&self) -> &'static str {
         "gcp_stackdriver_metrics"
     }
+
+    fn can_acknowledge(&self) -> bool {
+        false
```

Comment:
And here.

Reply:
On examination, it looks like all older `HttpSink` sinks should support this, so I'll fix those up.